commercionetwork/chain.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "commercionetwork",
+  "chain_id": "commercio-3",
+  "pretty_name": "Commercio.network",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "did:com:",
```

Comment:
Is this a valid bech32 path?

Reply:
it's been registered to SLIP173 a while

Reply:
@sunnya97 Yes, is a valid path. As specified in https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#Bech32 the range of characters allowed for hrp should also include “:”

Reply:
Wow, TIL